src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowHistoryV2Spec.groovy

```diff
@@ -298,6 +306,42 @@ class FlowHistoryV2Spec extends HealthCheckSpecification {
         ]
     }
 
+    @Tidy
+    @Tags(LOW_PRIORITY)
+    def "Info about a terminating switch is registered in flow history while rerouting"() {
```

Comment:
This is a bit misleading description. We should not really try to reroute a flow when terminating switch is down. We just change its status to DOWN without attempting to find a new path. Technically it's a part of 'reroute' process, but practically we are not even trying to find a new 'route'